src/Exception/ConfigNotSetException.php

```diff
@@ -0,0 +1,31 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Copyright 2010 - 2019, Cake Development Corporation (https://www.cakedc.com)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright Copyright 2010 - 2018, Cake Development Corporation (https://www.cakedc.com)
+ * @license MIT License (http://www.opensource.org/licenses/mit-license.php)
+ */
+
+namespace CakeDC\Users\Exception;
+
+use Cake\Core\Exception\Exception;
+
+class ConfigNotSetException extends Exception
+{
+    /**
+     * ConfigNotSetException constructor.
+     *
+     * @param array|string $message message
+     * @param int $code code
+     * @param null $previous previous
+     */
+    public function __construct($message, $code = 500, $previous = null)
```

Comment:
Extend from an existing 500 exception, then you don't need to declare a constructor at all